satip/utils.py

```diff
@@ -36,6 +36,10 @@
 from satip.scale_to_zero_to_one import ScaleToZeroToOne, compress_mask
 from satip.serialize import serialize_attrs
 
+# Assigning secrets from the .env file
+STORAGE_BACKEND_TOKEN = os.getenv('STORAGE_BACKEND_TOKEN')
```

Comment:
I would remove this, and just use the `os.getenv()` below

Reply:
> I would remove this, and just use the `os.getenv()` below

I've made this change in my latest commit